README.md

```diff
@@ -87,7 +88,7 @@ git - git should be installed in local machine. git is necessary for cloning thi
 
 Clone this repository to your desired folder:
 
-git clone https://github.com/Uchenna-ibute/Awesome-book-es6.git
+git clone https://https://github.com/Uchenna-ibute/To-do-list.git
 
 ### Install
```

Comment:
- [ ] To make your ReadMe file more professional write to commands to `install`, `run` your app and also to run the `tests`. You just have the headings with no commands. Kindly add them